junebug/channel.py

```diff
@@ -193,9 +193,17 @@ def _transport_config(self):
 
     @property
     def _application_config(self):
+        redis_manager = self.redis._config
+        # For tests, FAKE_REDIS cannot be an object, else deepcopy doesn't
+        # work.
+        if 'FAKE_REDIS' in redis_manager:
+            redis_manager['FAKE_REDIS'] = 'yes'
```

Comment:
Would we maybe be able to get away with not needing this because of [this](https://github.com/praekelt/vumi/blob/develop/vumi/persist/redis_base.py#L70-L72)? Or did you come across an issue before adding these lines?